core/src/main/java/lucee/runtime/tag/Http.java

```diff
@@ -1426,7 +1426,7 @@ else if(strPath != null) {
 	private void ssl(HttpClientBuilder builder) throws PageException {
 		try {
 			// SSLContext sslcontext = SSLContexts.createSystemDefault();
-			SSLContext sslcontext = SSLContext.getInstance("TLSv1.2");
+			SSLContext sslcontext = SSLContext.getInstance("TLS");
```

Comment:
Do we need this change? This will require doo amount of testing to make sure all outbound calls are working.